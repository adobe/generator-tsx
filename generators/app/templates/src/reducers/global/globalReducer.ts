import { types } from 'actions/global'
import createReducer from 'helpers/createReducer'

const defaultState = {}

export default createReducer('global', defaultState)(
	(state, action: types.GlobalActions) => {
		switch (action.type) {
			default:
				return state
		}
	},
)
