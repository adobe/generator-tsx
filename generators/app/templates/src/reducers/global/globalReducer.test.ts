import testReducer from 'utils/test/testReducer'

import globalReducer from './globalReducer'

testReducer(globalReducer, test => {
	test({ foo: 'VALUE' }, {} as any, { foo: 'VALUE' })
})
