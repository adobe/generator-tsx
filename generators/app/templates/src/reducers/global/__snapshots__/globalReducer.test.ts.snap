// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`globalReducer returns initial state 1`] = `Object {}`;
